shopfloor_mobile/static/wms/src/scenario/checkout.js

```diff
@@ -293,7 +293,17 @@ const Checkout = {
                         {path: "origin"},
                         {path: "carrier.name", label: "Carrier"},
                         {path: "move_line_count", label: "Lines"},
-                        {path: "priority", label: "Priority"},
+                        {
+                            path: "priority",
+                            render_component: "priority-widget",
+                            render_options: function (record) {
+                                const priority = parseInt(record.priority);
+                                // We need to pass the label to the component as an option instead of using "display_no_value"
```

Comment:
can't we prevent that instead? Empty stars or no star at all?

Reply:
I tried and I always got the "Priority:" label if there was no assigned priority, followed by no stars.
Displaying empty stars could work, but then it wouldn't be consistent with the way in which we display stars in the other scenario.

As a note, the customer has validated a screenshot that I shared with them here: https://jira.camptocamp.com/browse/BSCOS-4215

Reply:
Instead of adding a line, you could also have displayed the stars in-front of the title (picking name) like it is shown on odoo backend

Reply:
I tried that option but I thought it looked better this way. I think it's ok as is.